code/modules/admin/topic.dm

```diff
@@ -4583,3 +4584,259 @@
 			error_viewer.show_to(owner, locate(href_list["viewruntime_backto"]), href_list["viewruntime_linear"])
 		else
 			error_viewer.show_to(owner, null, href_list["viewruntime_linear"])
+
+	// ----- Religion and stuff
+	if (href_list["religions"])
+		#define MAX_MSG_LENGTH 200
+		#define NUMBER_MAX_REL 4
+		if (href_list["display"])
+			updateRelWindow()
+
+		switch (href_list["religions"])
+			if ("global_subtle_pm")
+				if (!href_list["rel"])
+					return FALSE
+
+				var/datum/religion/R = locate(href_list["rel"])
+
+				if (!istype(R, /datum/religion))
+					return FALSE
+
+				var/deity = sanitize(stripped_input(usr, "Which deity addresses this group of believers?", "Deity Name", R.deity_name), 1, MAX_NAME_LEN)
+				var/message = sanitize(stripped_input(usr, "Which message do you want to send?", "Message", ""), 1, MAX_MSG_LENGTH)
+
+				if (!deity || !message)
+					to_chat(usr, "<span class='warning'>Error: no deity or message selected.</span>")
+
+				for (var/datum/mind/M in R.adepts)
+					to_chat(M.current, "You hear [deity]'s voice in your head... <i>[message]</i>")
+
+				var/msg = "[key_name(usr)] sent message [message] to [R.name]'s adepts as [deity]"
+				msg = "<span class=\"admin\"><span class=\"prefix\">ADMIN LOG:</span> <span class=\"message\">[msg]</span></span>"
```

Comment:
Maybe I should have clarified: `message_admins` takes care of the whole formatting for you, you don't need this line anymore. Also you have more occurrences of this, and some other code still using `log_adminwarn`.